.cargo/config.toml

```diff
@@ -13,6 +13,10 @@ rustflags = [
     "-Clink-arg=-Wl,--no-rosegment", #4574
 ]
 
+# Remove this if your x86_64 macos does not support AVX2 vector extensions.
```

Comment:
Seems outdated. 😄 IIUC, we cannot compile without SIMD now?

Reply:
Since we removed the json parser based on `serde-json` and also removed the `allow-non-simd` feature of `simd-json`, we cannot compile to target without SIMD.

Reply:
yes, non-simd support is removed in #7259